Assets/LeapMotion/Scripts/LeapProvider.cs

```diff
@@ -145,7 +144,7 @@ public class LeapProvider :
       for (int searchHistoryIndex = 0; searchHistoryIndex < 60; searchHistoryIndex++) {
 
         leapMat = UnityMatrixExtension.GetLeapMatrix(this.transform);
```

Comment:
Last change I think, this recalculation of the leap matrix probably should be brought out of the loop.  Possible place it right before line 162?